styles/pypi/catppuccin.user.css

```diff
@@ -0,0 +1,1025 @@
+      @svg: escape(@raw_svg);
+
+      height: 185.85px;
+      width: 300px;
+      background-image: url("data:image/svg+xml;utf8,@{svg}");
+      background-repeat: no-repeat;
+      background-position-x: left;
```

Comment:
Just to be sure, this is styling an `img` element right?

Reply:
at the time of writing this i didn't look through the `tips-and-tricks.md` file so i `hidden`ed the `<img>` and applied this to the parent element, which works, although i can do what's shown in `tips-and-tricks.md` and style the `<img>` itself